packages/ui/components/icon/src/icons.js

```diff
@@ -1,4 +1,17 @@
 import { singletonManager } from 'singleton-manager';
+import { lazyfyManager } from '../../helpers/src/lazyfyManager.js';
 import { IconManager } from './IconManager.js';
 
-export const icons = singletonManager.get('@lion/ui::icons::0.x') || new IconManager();
+/**
+ * @returns {IconManager}
+ */
+function getIconManager() {
+  if (!singletonManager.has('@lion/ui::icons::0.x')) {
+    const iconManager = new IconManager();
+    singletonManager.set('@lion/ui::icons::0.x', iconManager);
```

Comment:
This (registration by default) deviates a bit from previous behaviour. Not necessarily bad though, as we now reached parity with how we did `getLocalizeManager` (and we will release this as a minor anyway)